policyengine_canada/variables/gov/provinces/nb/tax/income/credits/age_amount/nb_age_amount_credit.py

```diff
@@ -0,0 +1,17 @@
+from policyengine_canada.model_api import *
+
+
+class nb_age_amount_credit(Variable):
+    value_type = float
+    entity = Person
+    label = "New Brunswick age amount credit"
+    definition_period = YEAR
+    defined_for = ProvinceCode.NB
+
+    def formula(person, period, parameters):
+        age = person("age", period)
+        income = person("individual_net_income", period)
+        p = parameters(period).gov.provinces.nb.tax.income.credits.age_amount
+        eligibility = age >= p.age_eligibility
```

Comment:
I think this is our convention
```suggestion
        eligible = age >= p.age_eligibility
```